drivers/holykell_hpt604.json

```diff
@@ -0,0 +1,10 @@
+{
+    "common": {
+        "words": 1,
+        "datatype": "uint",
```

Comment:
Just realized that `uint` isn't actually a valid datatype - you should include the expected number of bits. The valid options are here: https://github.com/ammpio/ammp-edge/blob/main/src/processor/process_reading.py#L17. Since it's 1 word it will be 16 bits I believe, but good to be explicit.

Reply:
Thanks for the link!